packages/clio/clio.pacscript

```diff
@@ -0,0 +1,24 @@
+pkgname="clio"
+gives="clio"
+repology=("project: ${gives}")
+pkgver="1.0"
+pkgdesc="A cool CLI tool that displays facts about Linux, Unix, and computers using bash."
+arch=('amd64' 'arm64')
```

Comment:
```suggestion
arch=('all')
```

Since this is a bash script, this script works on *all* systems (provided they support bash).